File: 백준/8단계/손익분기점.js
const fs = require('fs')
let input = fs.readFileSync('/dev/stdin').toString().split(' ')

let A = Number(input[0])
let B = Number(input[1])
let C = Number(input[2])

let BEP = 0
if (C > B) {
    BEP = Math.floor(A / (C - B)) + 1
} else {
    BEP = -1
}

console.log(BEP)

File: 백준/8단계/손익분기점.py
A, B, C = map(int, input().split(' '))

# A 고정비용 B 가변비용 C 노트북 가격
# N * C > A + B * N(노트북 대수)
if C > B:
    print(int(A // (C - B) + 1))
else:
    print(-1)
